core/src/cats/dml/create_queryfiles.sh

```
@@ -86,10 +86,10 @@ for query in `ls ????_* | sed 's#\..*##g' | sort | uniq`; do
         if  [ -e "$query.$db" ]; then
             queryfile="$query.$db"
         fi
-        printf "/* %s */\n" "$queryfile" >> $queryincludefile
-        # remove comments and empty lines, add quotes on each line
-        cat "$queryfile" | sed -r -e "/^#/d" -e "/^$/d" -e 's/^(\s*)/\1"/' -e 's/\s*$/ "/' >> $queryincludefile
-        printf ',\n\n' >> $queryincludefile
+        printf '/* %s */\nR"SQL(' "$queryfile" >> $queryincludefile
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)